app/models/miq_widget.rb

```diff
@@ -108,6 +108,7 @@ def queue_generate_content_for_users_or_group(*args)
     MiqQueue.create_with(callback).put_unless_exists(
       :queue_name  => "reporting",
       :role        => "reporting",
+      :zone        => nil, # any zone
```

Comment:
Note, I didn't change this to use `submit_job` because that method doesn't support `put_unless_exists`.  Therefore, I have to set the zone to nil.